submission/submission-core/src/main/kotlin/ac/uk/ebi/biostd/submission/service/DoiService.kt

```diff
@@ -121,16 +122,20 @@ class DoiService(
         }
 
         organizations
-            .ifEmpty { throw MissingDoiFieldException(ORG_TYPE) }
+            .ifEmpty { throw MissingDoiFieldException(ORG_TYPE_1) }
             .forEach(::validate)
     }
 
+    private fun Section.findAttr(attribute: String): String? =
+        attributes.find { it.name.lowercase() == attribute.lowercase() }?.value
+
     companion object {
         internal const val AFFILIATION_ATTR = "Affiliation"
         internal const val NAME_ATTR = "Name"
         internal const val ORCID_ATTR = "ORCID"
 
-        internal const val ORG_TYPE = "Organization"
+        internal const val ORG_TYPE_1 = "Organization"
```

Comment:
rather that defined constant with capaital and then calling lowercase I think is simpler and efficient just have constant in lower case